p2p/src/authenticated/actors/tracker/address_record.rs

```diff
@@ -0,0 +1,86 @@
+use crate::authenticated::types::SignedPeerInfo;
+use commonware_cryptography::Scheme;
+use std::net::SocketAddr;
+
+/// If the count for an `AddressRecord` is set to this value,
+/// it is considered pinned and cannot be incremented or decremented.
+const PINNED: usize = usize::MAX;
```

Comment:
I suspect there is a way we can avoid using my original hack (maybe use Option<usize> for count in Unknown/Discovered?)

Reply:
Modified the logic here https://github.com/commonwarexyz/monorepo/pull/638/commits/867ad02d793411d943fb5227186144eace825556

Using `Option` would also be reasonable (I think just in `Discovered`; `Unknown` would always have a count)